### ISSUE TITLE
Enter is not fired for child button using tab: NVDA

### ISSUE THREAD

**Issue by ghost** [nvaccess/nvda#5981]
https://jsfiddle.net/8b58aL6m/

I am running NVDA software. I have a parent button and child button.
When I move to child button using tab and enter, the click event of parent is fired (Click of child is not fired at all). What is the reason for it and how can I fire enter of child button?

Everything works fine when I am not running NVDA software

**Comment by jcsteh** [comment]
What browser are you testing with?

**Comment by iyogeshsingh** [comment]
I am using chrome on Windows 10

**Comment by ehollig** [comment]
This appears to still be an issue with the latest version of Chrome. It works as expected in Firefox.

**Comment by Adriani90** [comment]
It is still reproducible in Chrome 71.

**Comment by ObjectInSpace** [comment]
I'm not able to get any odd behavior in Chrome or Firefox. When I tab to the embedded cancel button and press enter, the "hello 2" dialog is displayed followed by "hello 1" in all browsers and screenreaders.

**Comment by Adriani90** [comment]
Given that we have not get any updates from the initial author since years, I am closing this as abandoned. I cannot reproduce it in any browser with NVDA last alpha.
